### PR TITLE
Update bug-report.yml to add screenshots field

### PR DESCRIPTION
**Pull request by muthuswamyopn**
Added a screenshots field

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -33,7 +34,14 @@ body:
       label: Logs
       description: Add any logs/error messages to help debug the issue.
     validations:
-      required: true
+      required: false
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Upload any screenshots to help debug the issue.
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.yml:41]: I noticed there's a lack of alignment on the kind of phrase stored in the `description` fields:

> 1. description : A clear and concise description of what were your expectations.
> 1. description : A clear and concise description of what actually happened.
> 1. description : List the steps to reproduce the issue.
> 1. description: Upload any screenshots to help debug the issue.

a. 1 and 2 are describing the field itself while 3 and 4 are telling the submitter what to do. To align with 3 and 4, we would say something like "Provide a clear and concise description..."
a. 1 needs better alignment with 2 w/r/t sentence structure. To align better, and assuming we do (a), we would say something like "Provide a clear and concise description of what you expected to happen."

Comment (muthuswamyopn) [reply .github/ISSUE_TEMPLATE/bug-report.yml:41]: Modified @danfowler . Thanks

**Comment by danfowler** [comment]
@muthuswamyopn please request reviews from @omise/maintainers

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -10,14 +11,14 @@ body:
     id: expectation
     attributes:
       label: Expected behavior
-      description: A clear and concise description of what were your expectations.
+      description: Provide a clear and concise description of what were your expectations.
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.yml]:
```suggestion
      description: Provide a clear and concise description of what you expected to happen.
```

File: .github/ISSUE_TEMPLATE/pr.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+name: Pull Request
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/pr.yml]: @muthuswamyopn Please add the PR template in a separate PR.

**Review by danfowler — CHANGES_REQUESTED** [review]
Please add the PR template in a separate PR.

File: .github/ISSUE_TEMPLATE/pr.yml (hunk 3 above)
Comment (danfowler) [reply .github/ISSUE_TEMPLATE/pr.yml]: And remove it from this PR.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 name: Issue report
 description: Report an issue
 labels: ["bug"]
+assignees: omise/maintainers
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: Remove.

**Comment by muthuswamyopn** [comment]
Will place a new request

**Pull request closed without merging**